elife-1234567890-v2/elife-1234567890-v2.xml

```diff
@@ -321,6 +331,13 @@
                 <kwd>Human</kwd>
                 <kwd>Mouse</kwd>
             </kwd-group>
+            <kwd-group kwd-group-type="badges">
+                <!-- Controlled vocabulary for badge values: Open Data, Open Materials, Prereigstered -->
+                <title>Badges</title>
+                <kwd><ext-link ext-link-type="uri" xlink:href="https://www.cos.io/initiatives/badges">Open Data</ext-link></kwd>
+                <kwd><ext-link ext-link-type="uri" xlink:href="https://www.cos.io/initiatives/badges">Open Materials</ext-link></kwd>
+                <kwd><ext-link ext-link-type="uri" xlink:href="https://www.cos.io/initiatives/badges">Prereigstered</ext-link></kwd>
+            </kwd-group>
```

Comment:
Note we still haven't got the full requirements from editorial here. It might be that we'll need to add a footnote, or similar somewhere as well. (See for example [here](https://openscience.bmj.com/content/bmjos/6/1/e100272.full.pdf) from BMJ - badges won't be in the PDF though).